bigbluebutton-html5/imports/api/users/server/handlers/userJoined.js

```diff
@@ -1,10 +1,20 @@
 import { check } from 'meteor/check';
-
+import { Meteor } from 'meteor/meteor';
+import Users from '/imports/api/users';
 import addUser from '../modifiers/addUser';
 
 export default function handleUserJoined({ body }, meetingId) {
   const user = body;
   check(user, Object);
+  const hasUsers = Users.find({
+    clientType: 'HTML5',
+    connectionStatus: 'online',
+    guest: false,
+    meetingId,
+  }).fetch().length > 0;
 
+  if (!hasUsers) {
+    Meteor.call('requestGuestUsers', meetingId, user.intId);
```

Comment:
Why are we using meteor calls from for server<->server calls?
I would expect

```
import requestGuestUsers from methods/requestGuestUsers.js
requestGuestUsers(meetingId, user.intId);
```

Technically it does the same as the Meteor.call but so far we have used these calls only for client->server with some identifiers passed